shared/global-errors/index.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+import React, {Component} from 'react'
+import {Box, Text, Icon, HOCTimers} from '../common-adapters'
+import {globalStyles, globalColors, globalMargins, transition} from '../styles'
+
+import type {Props} from './index'
+
+type Size = 'Closed' | 'Small' | 'Big'
+type State = {
+  size: Size,
+  cachedSummary: ?string,
```

Comment:
we keep the summary/details so when we close and wipe the values we don't see it disappear immediatley as its animating